universal-application-tool-0.0.1/conf/application.conf

```diff
@@ -94,8 +94,11 @@ measurement_id = ${?MEASUREMENT_ID}
 support_email_address = "CiviForm@seattle.gov"
 
 staging_program_admin_notification_mailing_list = "seattle-civiform-program-admins-notify@google.com"
+staging_program_admin_notification_mailing_list = ${?STAGING_ADMIN_LIST}
```

Comment:
praise: thanks for parametrizing this! we really should go through and fix these hard-coded Seattle-specific strings